core/templates/services/utils.service.ts

```diff
@@ -160,7 +160,7 @@ export class UtilsService {
       return '/';
     }
 
-    if (urlString.charAt(0) !== '/') {
+    if (urlString.charAt(0) !== '/' || urlString.charAt(1) === '/') {
```

Comment:
Does this assume the URL has at least 2 chars?

Reply:
This works when the URL is < 2, because in such cases `.charAt()` returns an empty string.